### ISSUE TITLE
create HDLA page and backend methods

### ISSUE THREAD

**Issue by lucasnoah** [lucasnoah/litmetricsfrontend#8]
(Hierarchical Dirlecht Process): requires training corpus, infers number of topics:

Chunksize defaults to 256 {batch size in Wang article: that’s the number of DOCUMENTS used in each

iteration, NOT a chunking at the document level}

kappa defaults to 1.0 = learning rate {exponential decay rate} {default 0.51 based on Online LDA paper}

{little kappa is “How quickly old information is forgotten”

tau defaults to 64 = slow down parameter {downweights earlier iterations}

Kappa {might be Cap “K”} defaults to 15 = second level truncation level [Number of topics]

T defaults to 150 = top level truncation level [truncation is MAX number of topics for a document]

alpha defaults to 1 = second level concentration

gamma defaults to 1 = first level concentration

eta defaults to 0.01 = the topic dirichlet

scale defaults to 1

Var_converge defaults to 0.0001

Switch for optimal ordering of topics
